### ISSUE TITLE
Editorial update to RFCs/draft-34 + rawInfo + general consistency

### ISSUE THREAD

**Issue by rmarx** [quicwg/qlog#174]
since qlog started early on in the quic/H3 design process, we have some leftovers from early drafts in the docs.

We've updated most things over time as they were changed, but some aspects (e.g., most of the quic recovery events mention draft-23/27, `stream_state_updated` mentions draft-23, `packet_sent` triggers refer draft-23/19, etc.)

We need to go through each event and match it to terminology used in the final versions of the documents. 
For most things, this would just require removing mentions of old drafts, but in some cases this might require further changes. 

Finally, many events in the quic/H3 docs incorrectly use the pre-draft-02 way of logging raw information (e.g., as bytes fields directly), instead of properly using the new `raw:RawInfo` type. This should be updated.

Subsumes #46.

**Comment by rmarx** [comment]
Related to #113 as well.

**Comment by rmarx** [comment]
Related to #129

**Comment by rmarx** [comment]
Related to #166

**Comment by rmarx** [comment]
As discussed during the editors' meeting, it makes sense to keep this work until when we do the major pass over the full documents when updating to the new Data Definition format (#143). It makes sense to check these things when going through all event definitions piece-by-piece.

**Comment by LPardue** [comment]
Is there any more work remaining here? If so, recommend creating new more-specific tickets. Either way time to close this monolith.

**Comment by rmarx** [comment]
Yes, there's definitely more work remaining here, e.g., references to old drafts in https://www.ietf.org/archive/id/draft-ietf-quic-qlog-quic-events-06.html#section-4.6 and https://www.ietf.org/archive/id/draft-ietf-quic-qlog-quic-events-06.html#section-5.5. 

Figuring out what's what and creating separate tickets is probably more work than just going through the doc and fixing stuff though... I'd rather do the latter ;)
